cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ShortestPathPlanningTest.scala

```diff
@@ -42,29 +43,36 @@ class ShortestPathPlanningTest extends DocumentingTest {
+    p(
+      """Planning shortest paths in Cypher can lead to different query plans depending on the predicates that need
         |to be evaluated. Internally, Neo4j will use a fast bidirectional breadth-first search algorithm if the
         |predicates can be evaluated whilst searching for the path. Therefore, this fast algorithm will always
         |be certain to return the right answer when there are universal predicates on the path; for example, when
         |searching for the shortest path where all nodes have the `Person` label, or where there are no nodes with
         |a `name` property.""".stripMargin)
-    p("""If the predicates need to inspect the whole path before deciding on whether it is valid or not, this fast
+    p(
+      """If the predicates need to inspect the whole path before deciding on whether it is valid or not, this fast
         |algorithm cannot be relied on to find the shortest path, and Neo4j may have to resort to using a slower
         |exhaustive depth-first search algorithm to find the path. This means that query plans for shortest path
         |queries with non-universal predicates will include a fallback to running the exhaustive search to find
         |the path should the fast algorithm not succeed. For example, depending on the data, an answer to a shortest
         |path query with existential predicates -- such as the requirement that at least one node contains the property
         |`name='Charlie Sheen'` -- may not be able to be found by the fast algorithm. In this case, Neo4j will fall back to using
         |the exhaustive search to enumerate all paths and potentially return an answer.""".stripMargin)
-    p("""The running times of these two algorithms may differ by orders of magnitude, so it is important to ensure
+    p(
+      """The running times of these two algorithms may differ by orders of magnitude, so it is important to ensure
         |that the fast approach is used for time-critical queries.""".stripMargin)
-    p("""When the exhaustive search is planned, it is still only executed when the fast algorithm fails to find any
+    p(
+      """When the exhaustive search is planned, it is still only executed when the fast algorithm fails to find any
         |matching paths. The fast algorithm is always executed first, since it is possible that it can find a valid
         |path even though that could not be guaranteed at planning time.""".stripMargin)
+    p(
+      """Please note that falling back to the exhaustive search may prove to be a very time consuming strategy in some
+        |cases; such as when there is no shortest path between two nodes.
+        |Therefore, in these cases, it is recommended to set `cypher.forbid_exhaustive_shortestpath` to `true`,
+        |as explained in <<operations-manual#config_cypher.forbid_exhaustive_shortestpath, Operations Manual -> Configuration settings>>""".stripMargin)
```

Comment:
@jjaderberg I have tried numerous times to make this link work, but having no luck. Please can you take over? Thanks!